src/sync/sync_id.rs

```diff
@@ -7,7 +7,7 @@ use str_macro::str;
 lazy_static! {
     static ref SESSION_ID: String = {
         let mut randoms: [u8; 4] = [0; 4];
-        uuid::make_random_numbers(&mut randoms);
+        uuid::make_random_numbers(&mut randoms).unwrap();
```

Comment:
We could make `SESSION_ID` a `Result` if we want to be really careful.

Reply:
i'd just as well ignore the result here and let it default to 0000
```
        let _ = uuid::make_random_numbers(&mut randoms);
```